tests/test_tracking_cli.py

```diff
@@ -0,0 +1,169 @@
+import argparse
+import json
+import logging
+import os
+import shutil
+from pathlib import Path
+from tempfile import gettempdir
+from unittest.mock import call, patch
+
+import pytest
+from translations_parser.cli import experiments as experiments_publish
+from translations_parser.cli import taskcluster as tc_publish
+
+"""
+Tests tracking parser and publication via CLI entrypoints
+"""
+
+OUTPUT_DIR = Path(gettempdir()) / ".test_translations_parser_tmpdir"
```

Comment:
I believe we settled on using`data/tests` directory for temporary test data (see other tests).